api/server.go

```diff
@@ -6,15 +6,15 @@ import (
 	"net/http"
 	"strconv"
 
-	"github.com/gofrs/uuid"
-	"github.com/gorilla/mux"
-
 	"github.com/mattermost/mattermost-load-test-ng/cmd/loadtest/config"
 	"github.com/mattermost/mattermost-load-test-ng/loadtest"
 	"github.com/mattermost/mattermost-load-test-ng/loadtest/control"
 	"github.com/mattermost/mattermost-load-test-ng/loadtest/control/simplecontroller"
 	"github.com/mattermost/mattermost-load-test-ng/loadtest/store/memstore"
 	"github.com/mattermost/mattermost-load-test-ng/loadtest/user/userentity"
+
+	"github.com/gofrs/uuid"
```

Comment:
Also this is not needed anymore. We recently removed UUIDs.

Reply:
My branch still has them. I just moved it back to its original position (my editor moved it around...), so when we merge it'll be gone.